contents/blog/eu-cloud-hosting.md

```diff
@@ -0,0 +1,89 @@
+
+A stranger challenge was getting SSL certificates. CloudFront only supports certificates from `us-east-1` as the control plane is regional despite being a global service ([more details](https://aws.amazon.com/premiumsupport/knowledge-center/migrate-ssl-cert-us-east/)). This required some reconfiguring, but once we realized it, it was a quick change.
+
+Billing required a significant amount of work to get working on EU Cloud because it required different configurations for EU vs US. The test, production, EU, and US instances needed extra environment variables to configure. Settings were tweaked multiple times before it worked
+
+Other sticking points included getting AWS SES (Simple Email Service) approval, ensuring correct ClickHouse configuration, automating deployments, and more. 
+
+## Getting signups
+
+The driving force behind EU Cloud is customer demand. Before starting the feature, we already had companies interested. As it progressed, those companies made firmer commitments, mainly, being a part of the beta. In our [customer success](/handbook/small-teams/customer-success) team’s conversations, they continued to generate demand for EU Cloud.
+
+This built out a waitlist. To add to this, we launched a landing page for users to signup on their own. Any mention of EU Cloud before launch pointed at this landing page. 
+
+One important area of conversation was the scale of events EU Cloud could handle. The Customer Success and Infrastructure teams were in close communication to make sure the right expectations were set. Some PostHog users were expected to send a large number of events immediately after launch. Ensuring the infrastructure could handle it was critical. 
+
+## Final tests
+
+EU Cloud leverages the same tests NA Cloud does. As changes came into the core app, the team monitored that they worked well. For example, Team Infrastructure worked with Team Pipeline to ensure key migrations completed properly on EU Cloud. More work went into ensuring deployments were increasingly automated.
+
+Speed tests were made and found EU Cloud was significantly faster for EU customers than NA was (as expected). A basic speed test ran by our team found requests in the EU were ~70% faster to PostHog’s EU Cloud.
+
+Finally, before rolling out fully, the team tested with actual users over a few weeks before launch. They started with the smallest customer on the list and worked their way up to larger ones. When these users had no issues, the team knew they were good for launch.
+
+## Launch
+
+There was some final work from our website and docs team to create banners, the [landing page](/eu), and all the beautiful EU hedgehog art. Our marketing team worked on [blogs](/blog/posthog-cloud-eu), [tutorials](/tutorials/migrate-eu-cloud), emails, and more content for launch. We also made changes to the signup process, allowing users to pick their region during the signup process.
```

Comment:
```suggestion
There was some final work from our website and docs team to create banners, the [landing page](/eu), and all the beautiful EU hedgehog art. Our Marketing team worked on [blogs](/blog/posthog-cloud-eu), [tutorials](/tutorials/migrate-eu-cloud), emails, and more content for launch. We also made changes to the signup process, allowing users to pick their region during the signup process.
```

There's some inconsistency throughout on whether team names are capitalized or not. It's not a big deal either way (I don't think we have a rule on this in the style guide, even) -- but we should be consistent within a specific article.